### PR TITLE
remove ose_images.sh entirely from ose & ocp jobs

### PR DESCRIPTION
**Pull request by adammhaile**
@jupierce

### CODE REVIEW

**Comment by stevekuznetsov** [comment]
WHOA

**Comment by adammhaile** [comment]
@jupierce early latest removed.

**Comment by adammhaile** [comment]
@jupierce - refresh-images updated

**Comment by jupierce** [comment]
Depends on https://github.com/openshift/enterprise-images/pull/128

**Comment by adammhaile** [comment]
@jupierce  Removed OSE_REPO, as well as RHEL, GITHUB_BASE options as they were unused.

File: jobs/build/refresh-images/Jenkinsfile [hunk 1]
```diff
@@ -28,7 +20,7 @@ def mail_success() {
         subject: "Images have been refreshed: ${OSE_MAJOR}.${OSE_MINOR} ${OSE_GROUP}",
         body: """\
 Jenkins job: ${env.BUILD_URL}
-${OSE_MAJOR}.${OSE_MINOR}, Group:${OSE_GROUP}, Repo:${OSE_REPO}
+${OSE_MAJOR}.${OSE_MINOR}, Group:${OSE_GROUP}
```
Comment (jupierce) [inline jobs/build/refresh-images/Jenkinsfile]: Looks like OSE_GROUP is dead code as well.

**Comment by adammhaile** [comment]
@jupierce OSE_GROUP removed.

**Pull request merged**